docs/Tutorials/Contracts/Deploying-Contracts.md

```diff
@@ -155,18 +158,6 @@ Pass the following parameters to the
 to EthSigner; EthSigner then converts the request to an
 [`eth_sendRawTransaction`](../../Reference/API-Methods.md#eth_sendrawtransaction) call that Besu uses:
 
-```json
-params: {
-  "from": "0x9b790656b9ec0db1936ed84b3bea605873558198",
-  "to": null,
-  "gas": "0x76c0",
-  "gasPrice": "0x9184e72a000",
-  "data": "0x608060405234801561001057600080fd5b5060405161014d38038061014d8339818101604052602081101561003357600080fd5b8101908080519060200190929190505050806000819055505060f38061005a6000396000f3fe6080604052348015600f57600080fd5b5060043610603c5760003560e01c80632a1afcd914604157806360fe47b114605d5780636d4ce63c146088575b600080fd5b604760a4565b6040518082815260200191505060405180910390f35b608660048036036020811015607157600080fd5b810190808035906020019092919050505060aa565b005b608e60b4565b6040518082815260200191505060405180910390f35b60005481565b8060008190555050565b6000805490509056fea2646970667358221220e6966e446bd0af8e6af40eb0d8f323dd02f771ba1f11ae05c65d1624ffb3c58264736f6c63430007060033"
-}
-```
-
-The parameters are:
-
 * `to` - address of the receiver. To deploy a contract, set to `null`.
 * `from` - address of the EthSigner account. For example `0x9b790656b9ec0db1936ed84b3bea605873558198`.
```

Comment:
s/EthSigner/sender
it does have to be configured in EthSigner but that would be covered in the EthSigner setup docs